proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,1047 @@
+- [x] [`aria-valuenow`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow) is the same as `accessibilityValue.now`.
+- [x] [`aria-valuetext`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext) is the same as `accessibilityValue.text`.
+- [ ] ([`data-*`]() props. Resolves proposal [#323](https://github.com/react-native-community/discussions-and-proposals/issues/323))
+- [ ] [`dir`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) configures element's layout / writing direction.
+- [ ] [`elementTiming`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/elementtiming).
+- [ ] [`lang`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) identifies the language of the text content.
+- [ ] [`hidden`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) is the same as `display:'none'`.
+- [x] [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) is the same as `nativeID`.
+- [ ] [`inert`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert).
+- [x] [`role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) is the same as `accessibilityRole` (with support for web values).
+- [x] `style` is unchanged from existing React Native.
+- [x] [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) is the same as `focusable` (Android, Desktop). Add support for `0` and `-1` values only.
+
+## Common events
+
+Events supported on all elements.
+
+Each event handler should receive a W3C event object of the corresponding type, not a synthetic event. These props are merely conveniences and otherwise identical to using the `EventTarget` API mentioned later on. Note that no capture phase props are included, as listening to the capture phase (as well as custom events) can be handled with the `EventTarget` API. (Alternatively, we introduce `oncapture-click`, etc.)
+
+To avoid breaking changes in React Native and React DOM, it may be preferrable to create new event prop names that receive events as specified by web standards, e.g., `on-click`, `on-pointerup`.
+
+Resolves proposal [#492](https://github.com/react-native-community/discussions-and-proposals/issues/492).
+
+- [ ] `on-auxclick` is a `PointerEvent` for auxillary clicks.
+- [ ] `on-blur`.
+- [ ] `on-click` is a `PointerEvent` for primary clicks.
+- [ ] `on-contextmenu` is a `PointerEvent` for context clicks / long press.
+- [ ] `on-copy`.
+- [ ] `on-cut`.
+- [ ] `on-focus`.
+- [ ] `on-gotpointercapture`.
+- [ ] `on-lostpointercapture`.
+- [ ] `on-paste`.
+- [ ] `on-pointercancel`.
+- [ ] `on-pointerdown`.
+- [ ] `on-pointerenter`.
+- [ ] `on-pointerleave`.
+- [ ] `on-pointermove`.
+- [ ] `on-pointerout`.
+- [ ] `on-pointerover`.
+- [ ] `on-pointerup`.
+- [ ] `on-keydown`.
+- [ ] `on-keyup`.
+
+## `<a>` props
```

Comment:
Perhaps the most critical difference between web and native is how you can use strings as children in any component (used heavily in conjunction with the anchor element). Would be good to know where this project stands on adding that functionality to native.

Reply:
Auto-wrapping strings might be an option. More complicated is breaking the expectation that you can set text styles on random elements far up the tree and expect them to be inherited by elements all the way down. I suspect that isn't going to be on the cards.

Reply:
> More complicated is breaking the expectation that you can set text styles on random elements far up the tree and expect them to be inherited by elements all the way down.

I think inheriting text styles like that is unnecessary. Not crashing your entire app from `{"" && <div />}` already makes a big difference.